specification/migrate/resource-manager/readme.md

```diff
@@ -91,51 +119,60 @@ openapi-type: arm
 tag: package-2019-10
```

Comment:
If you are changing tag: package-2019-10 into package-migrate-2019-10, please also update the reference here.